src/framebuffer.rs

```diff
@@ -123,7 +123,9 @@ where
 {
     type Color = C;
 
-    // TODO: Optimise implementation by moving away from `ImageRaw::pixel` and directly calculating skip value.
+    /// Get pixel color.
```

Comment:
What is the benefit of adding a doc comment for this? Wouldn't it be enough to let rustdoc reuse the trait's doc comment?

If you want to keep the comment it should start with "Gets" or "Returns".

Reply:
Hm. I think this was from a botched rebase, but either way I've removed it as there is indeed no benefit.